app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php

```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
      * Usually this price is corresponding to price in admin panel of product
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
      */
     public function getRegularPrice();
 
     /**
      * Set regular price
      *
-     * @param string $regularPrice
+     * @param  string $regularPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
      */
     public function setRegularPrice($regularPrice);
 
     /**
      * Retrieve existing extension attributes object or create a new one.
      *
      * @return \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface|null
-     * @since 101.1.0
+     * @since  101.1.0
      */
     public function getExtensionAttributes();
 
     /**
      * Set an extension attributes object.
      *
-     * @param \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
+     * @param  \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
      * @return $this
-     * @since 101.1.0
+     * @since  101.1.0
      */
     public function setExtensionAttributes(
         \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
     );
-}
+}
```

Comment:
There has to be an empty line at the end of file.

Reply:
Fixed.